### ISSUE TITLE
Crash loop upon startup with "process crashed" popup

### ISSUE THREAD

**Issue by CNG** [hluk/CopyQ#1401]
**Describe the bug**
CopyQ shows an error popup and is apparently crashing and relaunching several times every second.

**To Reproduce**
Open CopyQ by any method, such as by running `copyq` in a console.

**Expected behavior**
CopyQ should "silently" open (without the error popup) and show an icon in the tray.

**Version, OS and Environment**
 - Application Version 3.11.1
 - OS Arch Linux, kernel 5.6.15-arch1-1
 - i3 window manager 4.18.1 (2020-04-23)

**Additional context**

I updated my Arch Linux packages today for the first time in about 3 days. After restarting, I noticed a CopyQ popup message about a crash and noticed my system logs show core dumps several times each second. I use Arch Linux and i3 window manager. I'm suspicious this is not caused by CopyQ itself and perhaps I need to rebuild something, but not sure how to determine that.

## Symptoms

I normally launch CopyQ via i3 window manager config at startup, but I have the same symptoms running `copyq` in the console.

I see the CopyQ icon in my tray and the popup at the lower right of my screen. Clicking the popup dismisses it, but a new one pops up immediately. I can still right click the CopyQ icon and open the settings. It seems to "stay open" despite the journal telling me it's crashing constantly.

The popup reads:

> **Command “copyq --clipboard-a...ss monitorClipboard”**  
> Error: Process crashed  
> 1. copyq --clipboard-access monitorClipboard

The first journal entries after launching CopyQ are:

```
Jun 03 13:05:28 Charlie18d audit[346713]: ANOM_ABEND auid=1000 uid=1000 gid=1002 ses=1 pid=346713 comm="copyq" exe="/usr/bin/copyq" sig=11 res=1
Jun 03 13:05:28 Charlie18d kernel: show_signal_msg: 9 callbacks suppressed
Jun 03 13:05:28 Charlie18d kernel: copyq[346713]: segfault at 0 ip 00007f147a94d8e0 sp 00007ffe594a18d8 error 4 in libQt5Gui.so.5.15.0[7f147a8d2000+4cd000]
Jun 03 13:05:28 Charlie18d kernel: Code: 10 4c 09 e8 49 89 40 04 31 c0 66 41 89 40 0c 48 83 c4 18 4c 89 c0 5b 5d 41 5c 41 5d 41 5e 41 5f c3 90 0f 1f 84 00 00 00 00 00 <48> 8b 06 8b 56 08 48 89 07 89 57 08 f0 83 00 01 c3 90 66 66 2e 0f
Jun 03 13:05:28 Charlie18d kernel: audit: type=1701 audit(1591214728.608:71690): auid=1000 uid=1000 gid=1002 ses=1 pid=346713 comm="copyq" exe="/usr/bin/copyq" sig=11 res=1
Jun 03 13:05:28 Charlie18d kernel: audit: type=1334 audit(1591214728.618:71691): prog-id=20460 op=LOAD
Jun 03 13:05:28 Charlie18d kernel: audit: type=1334 audit(1591214728.618:71692): prog-id=20461 op=LOAD
Jun 03 13:05:28 Charlie18d audit: BPF prog-id=20460 op=LOAD
Jun 03 13:05:28 Charlie18d audit: BPF prog-id=20461 op=LOAD
Jun 03 13:05:28 Charlie18d systemd[1]: Started Process Core Dump (PID 346723/UID 0).
Jun 03 13:05:28 Charlie18d audit[1]: SERVICE_START pid=1 uid=0 auid=4294967295 ses=4294967295 msg='unit=systemd-coredump@10219-346723-0 comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
Jun 03 13:05:28 Charlie18d kernel: audit: type=1130 audit(1591214728.618:71693): pid=1 uid=0 auid=4294967295 ses=4294967295 msg='unit=systemd-coredump@10219-346723-0 comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
Jun 03 13:05:28 Charlie18d systemd-coredump[346724]: Removed old coredump core.copyq.1000.26406ddd1a3d45acaa190ea4a7541e4a.222724.1591213147000000000000.lz4.
Jun 03 13:05:28 Charlie18d audit[346729]: ANOM_ABEND auid=1000 uid=1000 gid=1002 ses=1 pid=346729 comm="copyq" exe="/usr/bin/copyq" sig=11 res=1
Jun 03 13:05:28 Charlie18d kernel: copyq[346729]: segfault at 0 ip 00007fd2a443b8e0 sp 00007ffd53d58718 error 4 in libQt5Gui.so.5.15.0[7fd2a43c0000+4cd000]
Jun 03 13:05:28 Charlie18d kernel: Code: 10 4c 09 e8 49 89 40 04 31 c0 66 41 89 40 0c 48 83 c4 18 4c 89 c0 5b 5d 41 5c 41 5d 41 5e 41 5f c3 90 0f 1f 84 00 00 00 00 00 <48> 8b 06 8b 56 08 48 89 07 89 57 08 f0 83 00 01 c3 90 66 66 2e 0f
Jun 03 13:05:28 Charlie18d kernel: audit: type=1701 audit(1591214728.965:71694): auid=1000 uid=1000 gid=1002 ses=1 pid=346729 comm="copyq" exe="/usr/bin/copyq" sig=11 res=1
Jun 03 13:05:28 Charlie18d systemd-coredump[346724]: Process 346713 (copyq) of user 1000 dumped core.

                                                     Stack trace of thread 346713:
                                                     #0  0x00007f147a94d8e0 _ZN8QPaletteC1ERKS_ (libQt5Gui.so.5 + 0x15f8e0)
                                                     #1  0x00007f147a91d753 _ZN22QGuiApplicationPrivate10setPaletteERK8QPalette (libQt5Gui.so.5 + 0x12f753)
                                                     #2  0x00007f147a922915 _ZN22QGuiApplicationPrivate13updatePaletteEv (libQt5Gui.so.5 + 0x134915)
                                                     #3  0x00007f147a924712 _ZN22QGuiApplicationPrivate4initEv (libQt5Gui.so.5 + 0x136712)
                                                     #4  0x00007f147a925528 _ZN15QGuiApplicationC1ERiPPci (libQt5Gui.so.5 + 0x137528)
                                                     #5  0x000055d66ea3af9e _ZN11X11Platform34createClipboardProviderApplicationERiPPc (copyq + 0x20cf9e)
                                                     #6  0x000055d66e910fb5 _ZN15ClipboardClientC1ERiPPcRK11QStringListRK7QString (copyq + 0xe2fb5)
                                                     #7  0x000055d66e8f3868 main (copyq + 0xc5868)
                                                     #8  0x00007f1479dbf002 __libc_start_main (libc.so.6 + 0x27002)
                                                     #9  0x000055d66e8f422e _start (copyq + 0xc622e)

                                                     Stack trace of thread 346716:
                                                     #0  0x00007f1479e8d05f __poll (libc.so.6 + 0xf505f)
                                                     #1  0x00007f1479d5263b n/a (libxcb.so.1 + 0xc63b)
                                                     #2  0x00007f1479d5437b xcb_wait_for_event (libxcb.so.1 + 0xe37b)
                                                     #3  0x00007f1476ad2c51 n/a (libQt5XcbQpa.so.5 + 0x5dc51)
                                                     #4  0x00007f147a36ae0f n/a (libQt5Core.so.5 + 0xcde0f)
                                                     #5  0x00007f1479d27422 start_thread (libpthread.so.0 + 0x9422)
                                                     #6  0x00007f1479e97bf3 __clone (libc.so.6 + 0xffbf3)
Jun 03 13:05:28 Charlie18d audit[1]: SERVICE_STOP pid=1 uid=0 auid=4294967295 ses=4294967295 msg='unit=systemd-coredump@10219-346723-0 comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
Jun 03 13:05:28 Charlie18d kernel: audit: type=1131 audit(1591214728.972:71695): pid=1 uid=0 auid=4294967295 ses=4294967295 msg='unit=systemd-coredump@10219-346723-0 comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
Jun 03 13:05:28 Charlie18d kernel: audit: type=1334 audit(1591214728.978:71696): prog-id=20462 op=LOAD
Jun 03 13:05:28 Charlie18d kernel: audit: type=1334 audit(1591214728.978:71697): prog-id=20463 op=LOAD
Jun 03 13:05:28 Charlie18d audit: BPF prog-id=20462 op=LOAD
Jun 03 13:05:28 Charlie18d audit: BPF prog-id=20463 op=LOAD
Jun 03 13:05:28 Charlie18d audit[1]: SERVICE_START pid=1 uid=0 auid=4294967295 ses=4294967295 msg='unit=systemd-coredump@10220-346731-0 comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
Jun 03 13:05:28 Charlie18d systemd[1]: systemd-coredump@10219-346723-0.service: Succeeded.
Jun 03 13:05:28 Charlie18d systemd[1]: Started Process Core Dump (PID 346731/UID 0).
Jun 03 13:05:29 Charlie18d audit: BPF prog-id=20461 op=UNLOAD
Jun 03 13:05:29 Charlie18d audit: BPF prog-id=20460 op=UNLOAD
```

## Diagnostics

Here's some diagnostic output:

```console
$ copyq --version
CopyQ Clipboard Manager v3.11.1
Qt: 5.14.2
Compiler: GCC
Arch: x86_64-little_endian-lp64
OS: Arch Linux
```

```console
$ coredumpctl dump --reverse copyq
           PID: 347105 (copyq)
           UID: 1000 (cgorichanaz)
           GID: 1002 (cgorichanaz)
        Signal: 11 (SEGV)
     Timestamp: Wed 2020-06-03 13:05:32 PDT (6min ago)
  Command Line: /usr/bin/copyq --clipboard-access monitorClipboard
    Executable: /usr/bin/copyq
 Control Group: /user.slice/user-1000.slice/session-1.scope
          Unit: session-1.scope
         Slice: user-1000.slice
       Session: 1
     Owner UID: 1000 (cgorichanaz)
       Boot ID: 26406ddd1a3d45acaa190ea4a7541e4a
    Machine ID: 1c4c1ad094364de284c1d5e2bcb62eb4
      Hostname: Charlie18d
       Storage: /var/lib/systemd/coredump/core.copyq.1000.26406ddd1a3d45acaa190ea4a7541e4a.347105.1591214732000000000000.lz4
       Message: Process 347105 (copyq) of user 1000 dumped core.

                Stack trace of thread 347105:
                #0  0x00007fbd4fcd88e0 _ZN8QPaletteC1ERKS_ (libQt5Gui.so.5 + 0x15f8e0)
                #1  0x00007fbd4fca8753 _ZN22QGuiApplicationPrivate10setPaletteERK8QPalette (libQt5Gui.so.5 + 0x12f753)
                #2  0x00007fbd4fcad915 _ZN22QGuiApplicationPrivate13updatePaletteEv (libQt5Gui.so.5 + 0x134915)
                #3  0x00007fbd4fcaf712 _ZN22QGuiApplicationPrivate4initEv (libQt5Gui.so.5 + 0x136712)
                #4  0x00007fbd4fcb0528 _ZN15QGuiApplicationC1ERiPPci (libQt5Gui.so.5 + 0x137528)
                #5  0x0000562ebe3b2f9e _ZN11X11Platform34createClipboardProviderApplicationERiPPc (copyq + 0x20cf9e)
                #6  0x0000562ebe288fb5 _ZN15ClipboardClientC1ERiPPcRK11QStringListRK7QString (copyq + 0xe2fb5)
                #7  0x0000562ebe26b868 main (copyq + 0xc5868)
                #8  0x00007fbd4f14a002 __libc_start_main (libc.so.6 + 0x27002)
                #9  0x0000562ebe26c22e _start (copyq + 0xc622e)

                Stack trace of thread 347106:
                #0  0x00007fbd4f21805f __poll (libc.so.6 + 0xf505f)
                #1  0x00007fbd4f0dd63b n/a (libxcb.so.1 + 0xc63b)
                #2  0x00007fbd4f0df37b xcb_wait_for_event (libxcb.so.1 + 0xe37b)
                #3  0x00007fbd4be5dc51 n/a (libQt5XcbQpa.so.5 + 0x5dc51)
                #4  0x00007fbd4f6f5e0f n/a (libQt5Core.so.5 + 0xcde0f)
                #5  0x00007fbd4f0b2422 start_thread (libpthread.so.0 + 0x9422)
                #6  0x00007fbd4f222bf3 __clone (libc.so.6 + 0xffbf3)
Refusing to dump core to tty (use shell redirection or specify --output).
```

```console
$ ldd /usr/bin/copyq
        linux-vdso.so.1 (0x00007ffcbbdf2000)
        libXtst.so.6 => /usr/lib/libXtst.so.6 (0x00007efefcde1000)
        libX11.so.6 => /usr/lib/libX11.so.6 (0x00007efefcca0000)
        libQt5X11Extras.so.5 => /usr/lib/libQt5X11Extras.so.5 (0x00007efefcc99000)
        libQt5Network.so.5 => /usr/lib/libQt5Network.so.5 (0x00007efefcb18000)
        libQt5Svg.so.5 => /usr/lib/libQt5Svg.so.5 (0x00007efefcac1000)
        libQt5Script.so.5 => /usr/lib/libQt5Script.so.5 (0x00007efefc84e000)
        libQt5Widgets.so.5 => /usr/lib/libQt5Widgets.so.5 (0x00007efefc1b6000)
        libQt5Gui.so.5 => /usr/lib/libQt5Gui.so.5 (0x00007efefbae3000)
        libQt5Core.so.5 => /usr/lib/libQt5Core.so.5 (0x00007efefb592000)
        libstdc++.so.6 => /usr/lib/libstdc++.so.6 (0x00007efefb3b5000)
        libm.so.6 => /usr/lib/libm.so.6 (0x00007efefb270000)
        libgcc_s.so.1 => /usr/lib/libgcc_s.so.1 (0x00007efefb256000)
        libc.so.6 => /usr/lib/libc.so.6 (0x00007efefb08d000)
        libXext.so.6 => /usr/lib/libXext.so.6 (0x00007efefb078000)
        libXi.so.6 => /usr/lib/libXi.so.6 (0x00007efefb065000)
        libxcb.so.1 => /usr/lib/libxcb.so.1 (0x00007efefb03b000)
        libdl.so.2 => /usr/lib/libdl.so.2 (0x00007efefb035000)
        libpthread.so.0 => /usr/lib/libpthread.so.0 (0x00007efefb013000)
        libz.so.1 => /usr/lib/libz.so.1 (0x00007efefaff7000)
        libgssapi_krb5.so.2 => /usr/lib/libgssapi_krb5.so.2 (0x00007efefafa2000)
        libssl.so.1.1 => /usr/lib/libssl.so.1.1 (0x00007efefaf12000)
        libcrypto.so.1.1 => /usr/lib/libcrypto.so.1.1 (0x00007efefac35000)
        libGL.so.1 => /usr/lib/libGL.so.1 (0x00007efefabaf000)
        libpng16.so.16 => /usr/lib/libpng16.so.16 (0x00007efefab75000)
        libharfbuzz.so.0 => /usr/lib/libharfbuzz.so.0 (0x00007efefaa8c000)
        libmd4c.so.0 => /usr/lib/libmd4c.so.0 (0x00007efefaa76000)
        libsystemd.so.0 => /usr/lib/libsystemd.so.0 (0x00007efefa9cb000)
        libdouble-conversion.so.3 => /usr/lib/libdouble-conversion.so.3 (0x00007efefa9b4000)
        libicui18n.so.67 => /usr/lib/libicui18n.so.67 (0x00007efefa6b4000)
        libicuuc.so.67 => /usr/lib/libicuuc.so.67 (0x00007efefa4cd000)
        libpcre2-16.so.0 => /usr/lib/libpcre2-16.so.0 (0x00007efefa440000)
        libzstd.so.1 => /usr/lib/libzstd.so.1 (0x00007efefa36e000)
        libglib-2.0.so.0 => /usr/lib/libglib-2.0.so.0 (0x00007efefa245000)
        /lib64/ld-linux-x86-64.so.2 => /usr/lib64/ld-linux-x86-64.so.2 (0x00007efefd18f000)
        libXau.so.6 => /usr/lib/libXau.so.6 (0x00007efefa240000)
        libXdmcp.so.6 => /usr/lib/libXdmcp.so.6 (0x00007efefa238000)
        libkrb5.so.3 => /usr/lib/libkrb5.so.3 (0x00007efefa150000)
        libk5crypto.so.3 => /usr/lib/libk5crypto.so.3 (0x00007efefa11f000)
        libcom_err.so.2 => /usr/lib/libcom_err.so.2 (0x00007efefa119000)
        libkrb5support.so.0 => /usr/lib/libkrb5support.so.0 (0x00007efefa10a000)
        libkeyutils.so.1 => /usr/lib/libkeyutils.so.1 (0x00007efefa103000)
        libresolv.so.2 => /usr/lib/libresolv.so.2 (0x00007efefa0e9000)
        libGLdispatch.so.0 => /usr/lib/libGLdispatch.so.0 (0x00007efefa030000)
        libGLX.so.0 => /usr/lib/libGLX.so.0 (0x00007efef9ffd000)
        libfreetype.so.6 => /usr/lib/libfreetype.so.6 (0x00007efef9f29000)
        libgraphite2.so.3 => /usr/lib/libgraphite2.so.3 (0x00007efef9f04000)
        librt.so.1 => /usr/lib/librt.so.1 (0x00007efef9ef9000)
        liblzma.so.5 => /usr/lib/liblzma.so.5 (0x00007efef9ed1000)
        liblz4.so.1 => /usr/lib/liblz4.so.1 (0x00007efef9ead000)
        libgcrypt.so.20 => /usr/lib/libgcrypt.so.20 (0x00007efef9d8f000)
        libicudata.so.67 => /usr/lib/libicudata.so.67 (0x00007efef8278000)
        libpcre.so.1 => /usr/lib/libpcre.so.1 (0x00007efef8206000)
        libbz2.so.1.0 => /usr/lib/libbz2.so.1.0 (0x00007efef81f3000)
        libgpg-error.so.0 => /usr/lib/libgpg-error.so.0 (0x00007efef81ca000)
```

## Package updates

I've tried reinstalling via the Aurman "AUR helper" (aurman -S copyq), which installs the package using this configuration from
[PKGBUILD](https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/copyq):

```sh
pkgname=copyq
pkgver=3.11.1
pkgrel=1
pkgdesc="Clipboard manager with searchable and editable history"
url="https://github.com/hluk/${pkgname}"
depends=('hicolor-icon-theme' 'libxtst' 'qt5-script' 'qt5-svg' 'qt5-x11extras')
makedepends=('cmake' 'qt5-tools')
license=('GPL3')
arch=('x86_64')
source=("${pkgname}-${pkgver}.tar.gz::${url}/archive/v${pkgver}.tar.gz")
sha256sums=('a0b0a7d90aa67a48b4ab94b23bfa16da5602de80ef82862451bac619ede3d4bb')

build() {
    cd "CopyQ-${pkgver}"
    cmake -DWITH_WEBKIT=0 -DCMAKE_INSTALL_PREFIX=/usr .
    make
}

package() {
    cd "CopyQ-${pkgver}"
    make DESTDIR="${pkgdir}" install
}
```

I think this was likely triggered by an update to a package CopyQ depends on, because it looks like CopyQ itself was last updated after May 10 but before today, since I update all packages every few days:

```console
$ find /var/cache -name "copyq*" -ls 2>/dev/null
 24512588   1920 -rw-r--r--   1 root     root      1964042 Feb  2 05:54 /var/cache/pacman/pkg/copyq-3.10.0-1-x86_64.pkg.tar.zst
 24515849   1912 -rw-r--r--   1 root     root      1953899 May 10 10:27 /var/cache/pacman/pkg/copyq-3.11.1-1-x86_64.pkg.tar.zst
```

Looks like I last updated all packages June 1 and then again today, June 3.

```console
$ ack "aurman" ~/.zsh_history|colourify tail
: 1590808493:0;aurman -Syu --noconfirm
: 1590868510:0;aurman -Syu --noconfirm
: 1590871855:0;aurman -Ss paprefs
: 1590871861:0;aurman -S paprefs
: 1590962412:0;aurman -Syu --noconfirm
: 1590981732:488;aurman -Syu --noconfirm  # June 1
: 1590996359:11;aurman -Syu --noconfirm   # June 3
: 1591205009:524;aurman -Syu --noconfirm  # June 3
: 1591210292:9;aurman -Syu --noconfirm
: 1591215213:11;aurman -S copyq
```

Here's all the packages that were likely updated today, then:

```console
$ (cd /var/cache/pacman/pkg; find . -type f -newermt "$(date '+%Y-%m-%d %H:%M:%S' -d @1590996359)" 2>/dev/null) | sort
./arduino-builder-1.5.3-2-x86_64.pkg.tar.zst
./chromium-83.0.4103.61-2-x86_64.pkg.tar.zst
./convertlit-1.8-10-x86_64.pkg.tar.zst
./device-mapper-2.02.187-3-x86_64.pkg.tar.zst
./dhcpcd-9.1.0-1-x86_64.pkg.tar.zst
./docker-1:19.03.11-1-x86_64.pkg.tar.zst
./firefox-77.0.1-1-x86_64.pkg.tar.zst
./git-2.27.0-1-x86_64.pkg.tar.zst
./go-2:1.14.4-1-x86_64.pkg.tar.zst
./harfbuzz-2.6.7-1-x86_64.pkg.tar.zst
./harfbuzz-icu-2.6.7-1-x86_64.pkg.tar.zst
./haskell-hslua-1.1.1-1-x86_64.pkg.tar.zst
./haskell-hslua-module-system-0.2.1-40-x86_64.pkg.tar.zst
./haskell-hslua-module-text-0.2.1-49-x86_64.pkg.tar.zst
./haskell-ipynb-0.1.0.1-27-x86_64.pkg.tar.zst
./imagemagick-7.0.10.16-2-x86_64.pkg.tar.zst
./lcms2-2.10-1-x86_64.pkg.tar.zst
./libheif-1.7.0-1-x86_64.pkg.tar.zst
./librsvg-2:2.48.6-1-x86_64.pkg.tar.zst
./libtommath-1.2.0-3-x86_64.pkg.tar.zst
./libvirt-6.4.0-1-x86_64.pkg.tar.zst
./lvm2-2.02.187-3-x86_64.pkg.tar.zst
./nemo-4.6.2-1-x86_64.pkg.tar.zst
./pandoc-2.9.2.1-62-x86_64.pkg.tar.zst
./portaudio-1:19.6.0-7-x86_64.pkg.tar.zst
./python2-pyqt5-5.15.0-2-x86_64.pkg.tar.zst
./python2-pyqtwebengine-5.15.0-1-x86_64.pkg.tar.zst
./python2-sip-pyqt5-4.19.22-2-x86_64.pkg.tar.zst
./python-docker-4.2.1-1-any.pkg.tar.zst
./python-pyqt5-5.15.0-2-x86_64.pkg.tar.zst
./python-pyqt5-sip-12.8.0-1-x86_64.pkg.tar.zst
./python-pyqtwebengine-5.15.0-1-x86_64.pkg.tar.zst
./python-pytest-5.4.3-1-any.pkg.tar.zst
./qt5ct-0.41-4-x86_64.pkg.tar.zst
./re2-1:20200601-1-x86_64.pkg.tar.zst
./rubygems-3.1.3-1-any.pkg.tar.zst
./simple-scan-3.36.3-1-x86_64.pkg.tar.zst
./spandsp-0.0.6-3-x86_64.pkg.tar.zst
./strace-5.7-1-x86_64.pkg.tar.zst
./systemd-245.6-1-x86_64.pkg.tar.zst
./systemd-libs-245.6-1-x86_64.pkg.tar.zst
./systemd-sysvcompat-245.6-1-x86_64.pkg.tar.zst
./x86_energy_perf_policy-5.7-1-x86_64.pkg.tar.zst
./xapps-1.8.6-1-x86_64.pkg.tar.zst
./xkeyboard-config-2.30-1-any.pkg.tar.zst
./zsh-theme-powerlevel10k-1.11.0-1-x86_64.pkg.tar.zst
```

**Comment by davinellulinvega** [comment]
In addition to the list of likely updated packages in the previous post, Qt5 also had a pretty substantial update today. Here are the packages concerned:

```
qt5-base (5.14.2-3 -> 5.15.0-1)
python-pyqt5 (5.14.2-1 -> 5.15.0-2)
python2-sip-pyqt5 (4.19.22-1 -> 4.19.22-2)
python2-pyqt5 (5.14.2-1 -> 5.15.0-2)
qt5-declarative (5.14.2-1 -> 5.15.0-1)
qt5-graphicaleffects (5.14.2-1 -> 5.15.0-1)
qt5-location (5.14.2-2 -> 5.15.0-1)
qt5-multimedia (5.14.2-1 -> 5.15.0-1)
qt5-quickcontrols (5.14.2-1 -> 5.15.0-1)
qt5-quickcontrols2 (5.14.2-1 -> 5.15.0-1)
qt5-script (5.14.2-1 -> 5.15.0-1)
qt5-sensors (5.14.2-1 -> 5.15.0-1)
qt5-svg (5.14.2-1 -> 5.15.0-1)
qt5-tools (5.14.2-2 -> 5.15.0-2)
upgraded qt5-wayland (5.14.2-1 -> 5.15.0-1)
qt5-webchannel (5.14.2-1 -> 5.15.0-1)
qt5-webengine (5.14.2-3 -> 5.15.0-1)
qt5-webkit (5.212.0alpha4-4 -> 5.212.0alpha4-6)
qt5-websockets (5.14.2-1 -> 5.15.0-1)
qt5-x11extras (5.14.2-1 -> 5.15.0-1)
qt5ct (0.41-2 -> 0.41-4)
```
Hope this helps in any way.

**Comment by Lompik** [comment]
Can you try to disable qt5ct theme and run copyq ?
```sh
unset QT_QPA_PLATFORMTHEME
copyq
```
This fixes it for me.
I believe this is not a copyq issue but qt5ct one. Seems to be related to this https://sourceforge.net/p/qt5ct/tickets/77/

**Comment by davinellulinvega** [comment]
@Lompik This indeed did the trick for me as well. Thank you very much.

**Comment by j-degreef** [comment]
Yes, for me too.
Thanks

**Comment by stef204** [comment]
That's not really a fix though.  On my box, it removes the tray icon (disappears), etc.
Any plans to fix this issue in copyq itself?

**Comment by yichangshengwu** [comment]
> Can you try to disable qt5ct theme and run copyq ?
> 
> ```shell
> unset QT_QPA_PLATFORMTHEME
> copyq
> ```
> 
> This fixes it for me.
> I believe this is not a copyq issue but qt5ct one. Seems to be related to this https://sourceforge.net/p/qt5ct/tickets/77/

thanks, it works!
i commented QT_QPA_PLATHFORMTHEME=qt5ct in ~/.xprofile then copyq works

**Comment by hluk** [comment]
> Any plans to fix this issue in copyq itself?

It doesn't look like issue in CopyQ itself. I can only force unsetting QT_QPA_PLATFORMTHEME in the client (it's not needed there).

**Comment by CNG** [comment]
Ah good find @Lompik, thanks! CopyQ seems to work fine for me when I call it like `QT_QPA_PLATFORMTHEME= copyq`, and the icon still shows up for me unlike @stef204 .

Since this seems to be upstream issue in https://sourceforge.net/p/qt5ct/tickets/77/ and @hluk does not intend to address in CopyQ, probably I should close this issue here. Maybe it's worth keeping open for others coming here to be able to find the info more easily?

**Comment by hluk** [comment]
Thanks, for finding and sharing the workaround so quickly.

Let's keep it open for now.

I'm looking at Qt code if I can avoid it doing some unnecessary work like setting color palette for clipboard monitor process.

**Comment by hluk** [comment]
Hmm, CopyQ is already using `QGuiApplication::setDesktopSettingsAware(false)` so normally call to `QGuiApplication::setPalette()` is avoided. But if I set `QT_QPA_PLATFORMTHEME=qt5ct` it's called anyway from `Qt5CTPlatformTheme::applySettings()`. It's not crashing for me because I've used Qt v5.13.2 and qt5ct v0.38.

Maybe something unexpected is happening in new Qt when `desktopSettingsAware` is set and the `setPalette()` is called.

**Comment by Lompik** [comment]
I use ```qt5ct``` to modify ```copyq``` (and all qt apps) top menu style, font... it's been great at it. 
With this [commit](https://sourceforge.net/p/qt5ct/code/570/tree//trunk/qt5ct/src/qt5ct-qtplugin/qt5ctplatformtheme.cpp?diff=55111843c4d1040be5c80fcd:569) there are no more hiccups, we can continue to use both normally :)
Until the next ```qt5ct``` release, Arch users could use:
```bash
yay -S copyq-svn
```
Hopefully there will be a new release soon.

**Comment by Martina-Neumayer** [comment]
Thnx Lompik! Works for me superb. All I need was the qt5ct package update and reapplying my previously saved qt theming schema again.

**Comment by stef204** [comment]
> Arch users should just update, the patch has been included in the `0.41-5` version

@Lompik 
That works, thanks!
